permissions/plugin-kryptowire.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "kryptowire"
+github: "jenkinsci/kryptowire-plugin"
+paths:
+- "io/jenkins/plugins/kryptowire"
+developers:
+- "lrossett"
+- "mnairn"
+- "jasonmadigan"
+- "maleck13"
```

Comment:
Not a known user.

Reply:
I saw that but both of them have logged  in https://repo.jenkins-ci.org/webapp/#/home, does the json file takes a while to refresh itself?

Reply:
by json file I mean this file: http://reports.jenkins.io/reports/artifactory-ldap-users-report.json

Reply:
Generated periodically. We'll poke the PR builder in a few hours, and if it works, merge this.

Reply:
cool, thanks.